### DISCUSSION TITLE
Integrating react-native-wear-connectivity with renative

### DISCUSSION THREAD

**Discussion by fabOnReact** [pavjacko/renative#1651] [Ideas]
I'm the author of https://github.com/fabOnReact/react-native-wear-connectivity and I wanted to open a conversation about collaborating with renative to improve react-native/renative development with wearos.

- I would integrate my library react-native-wear-connectivity with renative
- I would adapt my work plans to better integrate with renative

https://www.renative.org/docs/plugins/overview#renative
I tested and developed apps for wearOS/Mobile using react-native. 
- What are the advantages of using renative over react-native for wearOS?
- Did you detect issues and fix issues with react-native development for wearOS? (based on the latest version, react-native supports wearOS development). 

Related https://github.com/flexn-io/renative/commit/f153b2fb3c036e8bd6d7ca7c979f029fff6dbfc9

Thanks a lot

**Comment by fabOnReact** [comment]
- [x] Test react-native-wear-connectivity with renative
- [x] Add entry in renative.plugin.json

**Reply by fabOnReact** [reply]
I tried installing the package. Seems to work. I'll update in a couple of days.

**Reply by fabOnReact** [reply]
https://github.com/user-attachments/assets/952c9592-7571-4dce-b5d2-98ef4beea2bb

**Comment by fabOnReact** [comment]
@pavjacko

I added the installation instructions for renative here https://github.com/fabOnReact/react-native-wear-connectivity?tab=readme-ov-file#installation-with-renative.

should I send a PR to add documentation to your website section Android Wear? Thanks  

https://next.renative.org/docs/platforms/androidwear

**Comment by pauliusguzas** [comment]
@fabOnReact Sorry for a bit late reply, it is nice to hear people using Renative with success. If you have anything you would like to add to docs or to Renative, you are welcome to create PR https://github.com/flexn-io/renative-docs or https://github.com/flexn-io/renative
